From 458ce42f8177cc51f882fc408d8252d30ece660c Mon Sep 17 00:00:00 2001
From: Donald Carr <sirspudd@gmail.com>
Date: Thu, 17 Nov 2016 15:04:43 -0800
Subject: [PATCH] Fix brcm-egl build by correcting commit() usage

Adjusted existing call to commit() in keeping with parallel changes in 5b807802

Change-Id: I31c17336efa9a79a5f6c1719702215db7239a97d
---
 src/hardwareintegration/client/brcm-egl/qwaylandbrcmeglwindow.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/hardwareintegration/client/brcm-egl/qwaylandbrcmeglwindow.cpp b/src/hardwareintegration/client/brcm-egl/qwaylandbrcmeglwindow.cpp
index 4d56d66..9842f61 100644
--- a/src/hardwareintegration/client/brcm-egl/qwaylandbrcmeglwindow.cpp
+++ b/src/hardwareintegration/client/brcm-egl/qwaylandbrcmeglwindow.cpp
@@ -252,9 +252,7 @@ void QWaylandBrcmEglWindow::swapBuffers()
         return;
 
     m_buffers[m_current]->bind();
-    attach(m_buffers[m_current], 0, 0);
-    damage(QRect(QPoint(), geometry().size()));
-    commit();
+    commit(m_buffers[m_current], QRegion(0, 0, geometry().size().width(), geometry().size().height()));
 
     m_current = (m_current + 1) % m_count;
     m_buffers[m_current]->waitForRelease();
-- 
2.10.2

